doc/tutorials/upgrading_configuration.rst

```diff
@@ -26,23 +26,27 @@ Scenario XML
 A scenario XML file captures the configurations of hypervisor features and
 definitions of VMs. Starting from v3.0, the highly recommended way to upgrade a
 scenario XML is to use the scenario XML upgrader found at
-``misc/config_tools/scenario_config/upgrader.py``. Invoke this script from the
+``misc/config_tools/scenario_config/upgrader.py``. Run this script from the
 command line in the following way:
 
-   .. code-block:: bash
+.. code-block:: bash
 
-      misc/config_tools/scenario_config/upgrader.py <your scenario XML> <out file>
+   misc/config_tools/scenario_config/upgrader.py <your scenario XML> <out file>
 
-``<your scenario XML>`` is the path to the scenario XML file for a previous
-release. This script will generate a new scenario XML file, located at ``<out
-file>``, by migrating, following the latest schema, as many data in ``<your
-scenario XML>`` as possible. If there is any data that cannot be migrated, the
-script will generate messages like this:
+* ``<your scenario XML>`` is the path to the scenario XML file for a previous
+  release.
 
-   .. code-block:: console
+* ``<out file>`` is the path to the new scenario XML file generated by the
+  script.
 
-      WARNING:root:hv/DEBUG_OPTIONS/some = '1' is discarded
-      INFO:root:vm[1]/os_config/name = 'OS Name': Guest OS names are no longer needed in scenario definitions.
+The script generates the new scenario XML file by migrating as many data in
+``<your scenario XML>`` as possible, based on the latest schema. If there is any
```

Comment:
You've been using ``data`` as plural previously and in the next sentence, so for consistency:
```suggestion
``<your scenario XML>`` as possible, based on the latest schema. If there are any
```